src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -323,6 +323,7 @@ winrt::Microsoft::Terminal::Settings::Model::CascadiaSettings CascadiaSettings::
     resultPtr->_ParseJsonString(DefaultJson, true);
     resultPtr->LayerJson(resultPtr->_defaultSettings);
     resultPtr->_ResolveDefaultProfile();

```

Comment:
I'm somewhat surprised that ... we can resolve the default profile before the profiles list is set up. Is this because default draws form _All_, but the menu draws from _Active_?

Reply:
Correct! This actually enables users to set a hidden profile as their default profile.

Reply:
neatoi.

Reply:
neato*